cli-tests/t_encrypt_raw_key.sh

```diff
@@ -32,7 +32,15 @@ head -c 32 /dev/urandom > "$raw_key_file"
 fscrypt encrypt --quiet --name=prot --source=raw_key --key="$raw_key_file" "$dir"
 show_status true
 
+begin "Encrypt with raw_key protector from stdin"
+head -c 32 /dev/urandom | fscrypt encrypt --quiet --name=prot --source=raw_key "$dir"
+show_status true
+
 begin "Try to encrypt with raw_key protector, using wrong key length"
 head -c 16 /dev/urandom > "$raw_key_file"
 _expect_failure "fscrypt encrypt --quiet --name=prot --source=raw_key --key='$raw_key_file' '$dir'"
 show_status false
+
+begin "Try to encrypt with raw_key protector from stdin, using wrong key length"
```

Comment:
Could you also add a test case which verifies that a directory can be encrypted using a raw_key file, then locked, and then unlocked using that raw_key from stdin --- or vice versa?  That would verify that the key gets processed correctly in both cases.

Reply:
Done.